core/serv/src/main.rs

```diff
@@ -95,6 +95,19 @@ impl CliArgs {
         self.data_dir.get_or_create()
     }
 
+    pub fn get_gsb_url(&self) -> Result<Url> {
```

Comment:
Shouldn't it default to `DEFAULT_GSB_URL` on Windows?

Reply:
Windows supports UNIX domain sockets and I wanted to use them on Windows as well. Unfortunately, `tokio::net` doesn't support it. There's [uds_windows](https://docs.rs/uds_windows/1.0.1/uds_windows/) but it doesn't seem async-compliant. 😞

Reply:
We'd need to limit the range of supported Windows versions first (it's 8.1 and 10 according to https://blog.golemproject.net/new-golem-alpha-ii-reveal-were-ready-for-you-to-come-build-with-us/). `AF_UNIX` support appeared in Windows 10 October 2018 update (https://github.com/Azure/mio-uds-windows#windows-support-for-unix-domain-sockets)

Reply:
Yep. But since I didn't find any asynchronous solution for Windows, I will just implement a fallback to TCP as you suggested.